src/servicex_did_finder_lib/servicex_adaptor.py

```diff
@@ -88,6 +94,32 @@ def put_file_add(self, file_info):
             self.logger.error(f'After {attempts} tries, failed to send ServiceX App a put_file '
                               f'message: {str(file_info)} - Ignoring error.')
 
+    def put_file_add_bulk(self, file_list):
+        for chunk in chunks(file_list, 20):
+            success = False
+            attempts = 0
+            mesg = []
+            for fi in chunk:
+                mesg.append({
+                    "timestamp": datetime.now().isoformat(),
```

Comment:
Duplicated code... can this be refactored to avoid two places where the json is created